services/director-v2/tests/unit/test_api_route_dynamic_scheduler.py

```diff
@@ -0,0 +1,160 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+import asyncio
+from typing import AsyncIterator
+
+import pytest
+import respx
+from fastapi import status
+from pytest import MonkeyPatch
+from pytest_mock.plugin import MockerFixture
+from pytest_simcore.helpers.typing_env import EnvVarsDict
+from simcore_service_director_v2.models.schemas.dynamic_services.scheduler import (
+    SchedulerData,
+)
+from simcore_service_director_v2.modules.dynamic_sidecar.errors import (
+    DynamicSidecarNotFoundError,
+)
+from simcore_service_director_v2.modules.dynamic_sidecar.scheduler import (
+    DynamicSidecarsScheduler,
+)
+from starlette.testclient import TestClient
+
+
+@pytest.fixture
+def mock_env(mock_env: EnvVarsDict, monkeypatch: MonkeyPatch) -> None:
+    monkeypatch.setenv("SC_BOOT_MODE", "default")
+    monkeypatch.setenv("DIRECTOR_ENABLED", "0")
+    monkeypatch.setenv("COMPUTATIONAL_BACKEND_ENABLED", "0")
+    monkeypatch.setenv("COMPUTATIONAL_BACKEND_DASK_CLIENT_ENABLED", "0")
+    monkeypatch.setenv("DIRECTOR_V2_TRACING", "null")
+
+    monkeypatch.setenv("DIRECTOR_V2_DYNAMIC_SCHEDULER_ENABLED", "true")
```

Comment:
minor: are you sure about having true here, and the other is 0 or 1? it just makes it a bit inconsistent